src/main/java/com/musala/sdcs/device/channel/base/AbstractChannel.java

```diff
@@ -32,10 +48,15 @@ public T getCommand() {
     private void setCommand(T command) {
         this.command = command;
     }
+    
+    public ChannelRepository getChannelRepository() {
```

Comment:
Why is this method not defined in the `GenericChannel` or the `Channel` interface (we should discuss).